### ISSUE TITLE
Realme 9 5G not unlocking

### ISSUE THREAD

**Issue by WalkerGTR** [turistu/rmx3474-rooting#6]
adb shell pm has-feature oppo.version.exp - true
adb shell getprop ro.product.name - RMX3474RU
adb shell getprop ro.product.model - RMX3474
adb shell getprop ro.build.version.ota - RMX3474_11.C.14_1140_202304031900

Still waiting for 2 days already...

**Comment by turistu** [comment]
@DysasterGates @aisivan36

Could you guys give it a try again?

if you still keep getting a -1006 answer, then first run the script with `cmd closeApply` and then apply again with `cmd applyLkUnlock`.

**Comment by DysasterGates** [comment]
Captured this moment on the video and it says(google translator)
![Screenshot_20230802_163832_com huawei himovie overseas](https://github.com/turistu/rmx3474-rooting/assets/104025916/405a26c2-dd5e-4086-82f4-798befa0d80b)
![изображение_2023-08-02_164534176](https://github.com/turistu/rmx3474-rooting/assets/104025916/38d7fe64-0ceb-49a0-9b4a-8488213824c8)

**Comment by turistu** [comment]
>  it said that my application is approved after 30 seconds without your script.

That's because you have submitted the application in the past, and they have finally approved it. Unfortunately, they haven't started supporting the phone model (neither `RMX3474RU`, `RMX3474EEA` or `RMX3474`).

But the phone did not accept the unlock code it got from the server, and did not enter fastboot mode (it should've shown a screeen like [this](https://cmuty-static-test.oss-cn-beijing.aliyuncs.com/CN/thread/1386910938103689216.png) if it had really entered fastboot mode). That's because, since the end of May, they have started including the model number in the unlock code. The people who applied with `cmd applyLkUnlock` before the end of May get a working unlock code, those who applied after get a garbage unlock code.

**tl;dr;** All this pretty much means that my script is pretty much useless with this model too ;-(

**Comment by DysasterGates** [comment]
So there is still no way to unlock bootloader?

**Comment by turistu** [comment]
Not now and not with this trick, sorry.

I suggest you [exit the deeptesting mode](https://static.c.realme.com/IN/wm-thread/1243536780746031104.png), since it only prevents you from applying for a new code with `applyLkUnlock` (should they ever revert that change to the server).

**Comment by DysasterGates** [comment]
We got a downgrade solution form A13 to A12. What chance, that your deeptestjunk trick will work on it?

**Comment by turistu** [comment]
I don't think it will work.

I haven't yet got the time to reverse engineer the bootloader, but it seems to do the same checks about the "new struct" and "model name".
